.github/workflows/build.yml

```diff
@@ -502,3 +502,50 @@ jobs:
 
       - name: Run jest tests
         run: yarn jest
+
+  i18n-tasks:
+    runs-on: ubuntu-22.04
+    services:
+      postgres:
+        image: postgres:10
+        ports: ["5432:5432"]
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_DB: open_food_network_test
+          POSTGRES_USER: ofn
+          POSTGRES_PASSWORD: f00d
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup redis
+        uses: supercharge/redis-github-action@1.4.0
+        with: 
+          redis-version: 6
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .node-version
+
+      - name: Install JS dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Set up database
+        run: |
+          bundle exec rake db:create
+          bundle exec rake db:schema:load
+      - name: specs on i18n-exceptions
+        run: bundle exec rspec ./spec/i18n-tasks/i18n_spec.rb
+        run: i18n-tasks missing
+        run: i18n-tasks unused
+        run: i18n-tasks check-consistent-interpolations
```

Comment:
The build is complaining, it seems you cannot have several `run` statements after each other

Reply:
Thanks @sigmundpetersen :-D

Reply:
I was trying to find a way to output the list of errors as well... So one could see immediately what the issue is, without having to run it locally.

Reply:
It seems the syntax the pipe is used above in the file, but still no output. I'll just remove it for now.